### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by srl295**
Bugs should be filed under the `full-icu` module (full-icu-npm repo). See https://github.com/unicode-org/full-icu-npm/pull/49

### CODE REVIEW

**Comment by srl295** [comment]
fyi @mylesborins

**Comment by srl295** [comment]
@mylesborins will CITGM want a new publish of this test when complete?

File: README.md [hunk 1]
```diff
@@ -24,4 +20,4 @@ Copyright © 1991-2016 Unicode, Inc. All rights reserved.
 Distributed under the Terms of Use in http://www.unicode.org/copyright.html.
 
 A copy of the ICU license is in [LICENSE.html](LICENSE.html)
-or is available at https://ssl.icu-project.org/repos/icu/icu/trunk/LICENSE
+or is available at [https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE](https://github.com/unicode-org/icu/blob/master/icu4c/LICENSE)
```
Comment (markusicu) [inline README.md:23]: why not let it auto-link?

Comment (srl295) [reply README.md:23]: > why not let it auto-link?

Autolink is a Github (GFM) extension, and not implemented everywhere. https://github.github.com/gfm/#autolinks-extension-

**Pull request merged**